dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/exchange/support/header/HeaderExchangeServer.java

```diff
@@ -210,14 +209,14 @@ public void reset(URL url) {
                 int h = url.getParameter(Constants.HEARTBEAT_KEY, heartbeat);
                 int t = url.getParameter(Constants.HEARTBEAT_TIMEOUT_KEY, h * 3);

```

Comment:
can be renamed to **urlTimeOutKey**. What do you say?

Reply:
These variable names are named by other people. In my opinion, stay the same is also OK.